spec/requests/api/v1/moves_controller_create_spec.rb

```diff
@@ -201,5 +201,43 @@
 
       it_behaves_like 'an endpoint that responds with error 422'
     end
+
+    context 'with a duplicate move', :skip_before do
+      let(:profile) { create(:profile) }
+      let(:person) { profile.person }
+      let(:move_attributes) do
+        attributes_for(:move).merge(date: old_move.date,
+                                    person: person,
+                                    from_location: from_location,
+                                    to_location: to_location)
+      end
+
+      before do
+        post '/api/v1/moves', params: { data: data, access_token: token.token }, as: :json
+      end
+
+      context 'when duplicate is cancelled' do
+        let!(:old_move) { create(:move, :cancelled, person: person, from_location: from_location, to_location: to_location) }
+
+        it_behaves_like 'an endpoint that responds with success 201'
+      end
+
+      context 'when duplicate is active' do
+        let!(:old_move) { create(:move, person: person, from_location: from_location, to_location: to_location) }
+        let(:errors_422) do
+          [
+            {
+              title: 'Unprocessable entity',
+              detail: 'Date has already been taken',
+              source: { 'pointer' => '/data/attributes/date' },
+              code: 'taken',
+              meta: { 'move_id' => old_move.id },
```

Comment:
Does this only support one active duplicate?

Reply:
given we allow duplicates, logically there can only be one duplicate.